### PR TITLE
feat(Web): add logout

### PR DESCRIPTION
**Pull request by julianalucena**
:bulb: Review per commit.

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ def validate_qrcode
     if web_otp_code
       SigninChannel.broadcast_to(
         form.signin_code, {
-          url: web_sessions_validate_otp_url,
+          url: validate_otp_web_session_url,
```
Comment (igorbpsimoes) [inline app/controllers/sessions_controller.rb:57]: Where does this come from?

File: spec/requests/web/sessions_spec.rb [hunk 2]
```diff
@@ -150,4 +150,20 @@
       end
     end
   end
+
+  describe "DELETE /destroy" do
+    subject(:dispatch_request) { delete "/web/session" }
+
+    it 'reset user session' do
```
Comment (hmenezes) [inline spec/requests/web/sessions_spec.rb:157]: :nail_care: 
```suggestion
    it 'resets user session' do
```

**Review by hmenezes — APPROVED** [review]

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (julianalucena) [reply app/controllers/sessions_controller.rb:57]: `config/routes.rb`

File: spec/requests/web/sessions_spec.rb (hunk 2 above)
Comment (julianalucena) [reply spec/requests/web/sessions_spec.rb:157]: faf513618ebe1433b648d4582aaf587a659762ba

**Review by igorbpsimoes — APPROVED** [review]

**Pull request merged**